docs/signals.rst

```diff
@@ -0,0 +1,25 @@
+Django Signals
+================
+
+**django-auth-adfs** uses Django `Signals <https://docs.djangoproject.com/en/stable/topics/signals/>`
+to allow the application to listen for and execute custom logic at certain points in the authentication
+process. Currently, the following signals are supported:
+
+* ``post_authenticate``: sent after a user has been authenticated through either the ``AdfsAuthCodeBackend``
+  or the ``AdfsAccessTokenBackend`` (and created in Django, if ``CREATE_NEW_USERS`` is enabled) and
```

Comment:
Ideally, we'd put a ref on CREATE_NEW_USERS in settings_ref.rst and use the `ref` syntax here.